Script/PRONTO.py

```diff
@@ -1455,42 +1511,29 @@ def main(argv):
 				slide6_table_height = 1.63
 				slide6_table_font_size = 7
 				if_print_rowNo = False
+				table6_column_width = []
```

Comment:
I am not sure if you need to define it if it is empty. Couldn't you just put the empty list in the function instead?